STATUS: INLINE SUCCESSFUL
MODIFIED: de.strullerbaumann.visualee.filter.entity.ClassFilterTest.testIsOkExcludeFalse
MODIFIED FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/ClassFilterTest.java
CLEANER: de.strullerbaumann.visualee.filter.entity.TestGroup100Case0.test253
CLEANER FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/TestGroup100Case0.java
POLLUTER: de.strullerbaumann.visualee.filter.entity.ClassFilterTest.testIsOkExcludeTrue
POLLUTER FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/ClassFilterTest.java
ORIGINAL CLEANER SIZE: 16
NEW CLEANER SIZE: 2
ELAPSED TIME: 109.54699993133545
==========================
@@ -102,0 +102,4 @@
+ {
+     de.strullerbaumann.visualee.filter.boundary.FilterContainer filterContainer0 = de.strullerbaumann.visualee.filter.boundary.FilterContainer.getInstance();
+     filterContainer0.clear();
+ }
